frontend/src/metabase/static-viz/components/XYChart/utils/ticks.ts

```diff
@@ -19,6 +19,7 @@ import {
   XValue,
 } from "metabase/static-viz/components/XYChart/types";
 import { getX } from "metabase/static-viz/components/XYChart/utils/series";
+import { max } from "d3";
```

Comment:
@alxnddr There are issues with importing `d3` like that. The thing is that you would be using deprecated `d3` version as newer versions are split into individual libraries, like `d3-scale`. That's why `visx` libs are using separate packages, and by importing `d3` this bundles 2 separate library versions.

Reply:
Thanks, good catch!